package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "insertdemo",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
```

Comment:
`index.js` not exists so update this too

Reply:
ok sir